test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpCodeGeneratorTest.cs

```diff
@@ -14,9 +17,12 @@ namespace Microsoft.AspNetCore.Razor.CodeGenerators
 {
     public class CSharpCodeGeneratorTest
     {
+        private static readonly string _testOutputDirectory = "TestFiles/CodeGenerator/Output/CSharpCodeGenerator";
```

Comment:
On second thought, leave the final folder there. Just make sure the files are written correctly w/ `GENERATE_BASELINES`.